src/EFCore.MySql/Migrations/MySqlMigrationsSqlGenerator.cs

```diff
@@ -391,16 +391,10 @@ protected override void Generate(DropSchemaOperation operation, IModel model, Mi
             builder
                 .Append("CREATE SEQUENCE ")
```

Comment:
`CREATE SEQUENCE` is only supported by some MariaDB versions as of now. Therefore, this methods needs to check the server version accordingly. Add the necessary support constructs to [ServerVersion.Support.cs](https://github.com/PomeloFoundation/Pomelo.EntityFrameworkCore.MySql/blob/master/src/EFCore.MySql/Storage/ServerVersion.Support.cs) and then use them to check that support exists here. For the general idea, take a look at the [Npgsql](https://github.com/npgsql/efcore.pg/blob/795cb6bdf789ae601c2a483178850c930fbc5d4b/src/EFCore.PG/Migrations/NpgsqlMigrationsSqlGenerator.cs#L1312-L1330) implementation (though our syntax is different when in comes to version checks).